grails-app/views/locationType/create.gsp

```diff
@@ -41,7 +41,6 @@
                             </td>
                             <td valign="top" class="value ${hasErrors(bean: locationTypeInstance, field: 'description', 'errors')}">
                                 <g:select name="locationTypeCode" class="chzn-select-deselect" from="${org.pih.warehouse.core.LocationTypeCode.list()}"
-                                          noSelection="['':'']"
```

Comment:
As a reminder, removing this line will default the selection to the first LocationTypeCode (which is DEPOT)